.travis.yml

```diff
@@ -22,6 +22,7 @@ before_script:
   - ./scripts/sauce/sauce_connect_block.sh
 
 script:
+  - node_modules/.bin/tslint -c tslint.json 'lib/**/*.ts' 'test/**/*.ts'
```

Comment:
Thank you for the review, you are right, there already has a task in gulp 'gulp lint', I will use that one.